source/reference/color_schemes.rst

```diff
@@ -65,9 +67,11 @@ by means of the **Preferences → Color Scheme** menu.
 A common way of selecting a color scheme
 is by associating it to a type of file
 using the file-type-specific settings.
-For example, for the Python file type (``syntax_file`` == :file:`Python.tmLanguage`),
-we'd use the :file:`Python.sublime-settings` file
-and, within, set ``color_scheme`` to some color scheme file.
+Select **Preferences → Settings - Syntax Specific**
+to open the settings file
```

Comment:
above we say `file-type-specific`settings and now it seems we talk about something different. Consider using `file-type-specific` again.